plugins/@grouparoo/airtable/src/lib/client/client.ts

```diff
@@ -0,0 +1,181 @@
+import Airtable, { FieldSet, Record, RecordData, Records } from "airtable";
+import { AirtableAppOptions } from "../appOptions";
+import Axios, { AxiosRequestHeaders, AxiosError } from "axios";
+import { URL } from "url";
+import {
+  CreateRecord,
+  HealthResponse,
+  ListTablesResponse,
+  Table,
+} from "./models";
+import { IClient } from "./interfaces/iClient";
+import AirtableError from "airtable/lib/airtable_error";
+
+export class Client implements IClient {
+  private readonly apiKey: string;
+  private readonly baseClient: Airtable.Base;
+  private readonly apiURL: URL;
+  private readonly baseId: string;
+  constructor(appOptions: AirtableAppOptions) {
+    this.apiKey = appOptions.apiKey;
+    this.baseClient = new Airtable({
+      apiKey: appOptions.apiKey,
+      endpointUrl: appOptions.airtableHost,
+    }).base(appOptions.baseId);
+    this.baseId = appOptions.baseId;
+    this.apiURL = new URL(appOptions.airtableHost);
+  }
+  private defaultHeaders(): AxiosRequestHeaders {
+    return {
+      Authorization: `Bearer ${this.apiKey}`,
+    };
+  }
+  async health(): Promise<HealthResponse> {
+    const healthEndpoint = new URL("/v0/meta/bases", this.apiURL);
+    return Axios.get<HealthResponse>(healthEndpoint.href, {
+      headers: this.defaultHeaders(),
+    })
+      .then((resp) => {
+        return {
+          body: resp.data,
+          statusCode: resp.status,
+        };
+      })
+      .catch((reason: AxiosError) => {
+        return {
+          body: reason.response.data,
+          statusCode: reason.response.status,
+        };
+      });
+  }
```

Comment:
This method is going to be a problem.  Right now, Airbase is not granting new API access tokens to the `meta` API.  So for me, I can't pass the health check.  It's a shame, because there's no way for me to test this right now.

The same problem exists for the `listTables()` API below.  At the very least, we need to note somewhere that this plugin requires [`meta` API access](https://airtable.com/shrWl6yu8cI8C5Dh3)